packages/@react-aria/landmark/test/useLandmark.test.tsx

```diff
@@ -130,13 +131,13 @@ describe('LandmarkManager', function () {
     let navigation = tree.getByRole('navigation');
     let main = tree.getByRole('main');
 
-    fireEvent.keyDown(document.activeElement, {key: 'F6'});
-    fireEvent.keyUp(document.activeElement, {key: 'F6'});
-    expect(document.activeElement).toBe(navigation);
+    fireEvent.keyDown(document.activeElement as HTMLElement, {key: 'F6'});
```

Comment:
why is that needed? `fireEvent` should accept any `Element` as the target.

Reply:
document.activeElement is defined as `Element | null`
and fireEvent takes `Element | Node | Document | Window`
:(

I could do something like `document.activeElement!` instead though?

Reply:
Ugh, why is document.activeElement defined to include null? I don't think that can ever happen in a real browser - it's always `document.body` or some other element right?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Document/activeElement#value

Reply:
https://html.spec.whatwg.org/multipage/interaction.html#dom-documentorshadowroot-activeelement-dev

Reply:
i don't know when null would ever be returned... but there is the reason for the typings

Reply:
so according to the steps there it could occur if there is both no `<body>` or `<html>` element which seems very unlikely haha

I guess `document.activeElement!` is preferable over a type cast but still annoying. that will affect basically every single test file for an extremely edge case.